### DISCUSSION TITLE
PCAP: No data available for the requested dates

### DISCUSSION THREAD

**Discussion by BustedSec** [Security-Onion-Solutions/securityonion#2969] [Unsupported Versions]
No matter what I ask of it - it tells me there is no data available. It also seems to be defaulting to putting 0 in for source port even though I have no entry in that field.

**Comment by weslambert** [comment]
Have you confirmed there is PCAP being written on the sensor node in /nsm/pcap?

**Reply by BustedSec** [reply]
alright, it appears there is not anything there. 
running so-status gives me 

Checking Docker status

    Docker --------------------------------------------------------------------------- [ OK ]

Checking container statuses

    so-cortex ------------------------------------------------------------------------ [ OK ]
    so-curator ----------------------------------------------------------------------- [ OK ]
    so-dockerregistry ---------------------------------------------------------------- [ OK ]
    so-elastalert -------------------------------------------------------------------- [ OK ]
    so-elasticsearch ----------------------------------------------------------------- [ OK ]
    so-filebeat ---------------------------------------------------------------------- [ OK ]
    so-fleet ------------------------------------------------------------------------- [ OK ]
    so-grafana ----------------------------------------------------------------------- [ OK ]
    so-idstools ---------------------------------------------------------------------- [ OK ]
    so-influxdb ---------------------------------------------------------------------- [ OK ]
    so-kibana ------------------------------------------------------------------------ [ OK ]
    so-kratos ------------------------------------------------------------------------ [ OK ]
    so-logstash ---------------------------------------------------------------------- [ OK ]
    so-mysql ------------------------------------------------------------------------- [ OK ]
    so-nginx ------------------------------------------------------------------------- [ OK ]
    so-playbook ---------------------------------------------------------------------- [ OK ]
    so-redis ------------------------------------------------------------------------- [ OK ]
    so-sensoroni --------------------------------------------------------------------- [ OK ]
    so-soc --------------------------------------------------------------------------- [ OK ]
    so-soctopus ---------------------------------------------------------------------- [ OK ]
    so-telegraf ---------------------------------------------------------------------- [ OK ]
    so-thehive ----------------------------------------------------------------------- [ OK ]
    so-thehive-es -------------------------------------------------------------------- [ OK ]
    so-wazuh ------------------------------------------------------------------------- [ OK ]

any idea what I'm missing?

**Reply by BustedSec** [reply]
oddly enough I am getting alerts despite their not being any pcaps.

**Reply by w4rc0n** [reply]
Sounds like suricata is getting pcap just fine and alerting on it, but stenographer isn't writing PCAPs

**Comment by BustedSec** [comment]
Got any stenographer troubleshooting tips?
